### DISCUSSION TITLE
New to Nuxt + SQLite

### DISCUSSION THREAD

**Discussion by belius** [nuxt/nuxt#30150] [Questions]
Hi,

I am implementing an API that needs to return data from a SQLite Database.

The structure of the server folder in my project is this:

--> server
------> api
----------> report.ts
------> db
----------> survey.db

in report.ts I have this code

```typescript
import sqlite3 from "sqlite3";

export default defineEventHandler(async (event) => {
    const db = new sqlite3.Database("../db/survey.db");
  
    .....
});
```

but when I run it I receive

```console
[ERROR] [nitro] [uncaughtException] SQLITE_CANTOPEN: unable to open database file
```

can you please help me to understand what I am doing wrong?

Thank you

**Comment by pi0** [comment]
Hi. Please take a look at [nitro sql](https://nitro.build/guide/database) docs, there is built-in support.

**Reply by belius** [reply]
> Hi. Please take a look at [nitro sql](https://nitro.build/guide/database) docs, there is built-in support.

Thank you.
I've seen it, but my use case has several databases that I have to put in a folder and I am wondering how to get the path of the db server side

Do you have any idea?
